### ISSUE TITLE
instanceMap

### ISSUE THREAD

**Issue by AndyDev25** [vuejs/vue-devtools#1331]
### Version
6.0.0-beta.2

### Browser and OS info
Chrome 87 / Windows 10

### Steps to reproduce
Add to mounted firebase.auth() and then if(firebase.auth()){ declare it }

### What is expected?
Clear console if i dont count Vue devtools classic 'Enabling backend for Vue'

### What is actually happening?
Clear console if i dont count Vue devtools classic 'Enabling backend for Vue'

After adding this to mounted: 
const user = fbauth.currentUser
    if (user) {
      this.showauth = false
      this.currentUser = user.displayName
      this.email = user.email
      this.getuid = user.uid
    }
(i had set fbauth to firebase.initializeApp(FireBaseConfig) and than firebaseApp.auth() ) 
it starts saying ->

Uncaught (in promise) TypeError: Cannot read property 'instanceMap' of null
    at Object.<anonymous> (backend.js:603)
    at Object.emit (<anonymous>:1:780)
    at eval (runtime-core.esm-bundler.js?5c40:552)
    at unmountComponent (runtime-core.esm-bundler.js?5c40:4767)
    at unmount (runtime-core.esm-bundler.js?5c40:4641)
    at patch (runtime-core.esm-bundler.js?5c40:3684)
    at patchBlockChildren (runtime-core.esm-bundler.js?5c40:4013)
    at patchElement (runtime-core.esm-bundler.js?5c40:3977)
    at processElement (runtime-core.esm-bundler.js?5c40:3795)
    at patch (runtime-core.esm-bundler.js?5c40:3712)

I've done same in Vue 2 and everything works fine.

**Comment by Silly-V** [comment]
How'd you fix it?

**Comment by nicosemp** [comment]
I still have this problem

**Comment by Silly-V** [comment]
I started getting this problem immediately when I began using rxjs network call cancellation with the `takeUntil` method. I restarted the page and re-ran `npm run serve` but the problem persisted. Then I went to chrome://extensions and hit the Update button on the Vue3 dev tools extensions. The version, I think was already the latest, but this forced some reconnection to the dev tools after which the problem went away! Hopefully this provides any clues to whomever is interested.
